.storybook/config.js

```diff
@@ -48,6 +42,8 @@ addDecorator(story => (
   /* eslint-enable react/jsx-filename-extension */
 ));
 
+addDecorator(story => <WithTimemachine>{story()}</WithTimemachine>);
```

Comment:
This Pr is titled `restrict timemachine to ArticleTimestamp stories`, is it doing that? This is a global decorator

Reply:
We initially started out as that, before realising a global decorator was possible. I'll change the PR title.